alerts/cloudtrail_excessive_describe.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# Copyright (c) 2014 Mozilla Corporation
+
+
+from lib.alerttask import AlertTask
+from mozdef_util.query_models import SearchQuery, TermMatch, ExistsMatch
+
+
+class AlertCloudtrailExcessiveDescribe(AlertTask):
+    def main(self):
+        # Create a query to look back the last 20 minutes
+        search_query = SearchQuery(minutes=20)
+
+        # Add search terms to our query
+        search_query.add_must([
+            TermMatch('source', 'cloudtrail'),
+            TermMatch('details.eventverb', 'Describe'),
+            ExistsMatch('details.source')
```

Comment:
It looks like using `details.source` as the aggregation field doesn't work. This field is only present in autoscaling events which probably isn't what was intended.

I'm not entirely sure what was envisioned for this alert, if it was to detect when a single threat actor was making many describe calls, or when a single AWS service (e.g. DynamoDB) was receiving many describe calls (regardless of who's making them), or when an AWS account as a whole is receiving many describe calls (again, unrelated to who's making them)

If I can figure out what the intent is I can suggest what field should be used for aggregation.

As it is right now this only detects when AWS autoscaling service makes too many describe calls.